WcaOnRails/db/migrate/20160731000001_create_committees.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class CreateCommittees < ActiveRecord::Migration
+  def change
+    create_table :committees do |t|
+      t.string :name, limit: 50, null: false
+      t.string :slug, limit: 50, null: false
```

Comment:
Hmm, we use variable for limit validation, what if we make it larger in the future (just wondering).
@larspetrus what's the best practice, is it worth using SQL limit?